<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form3</class>
 <widget class="QWidget" name="Form3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QCheckBox" name="softmusicBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Softmusic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Domyślna separacja panning:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="loopBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Odczytuj zapętlenia</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hqmixerBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Hqmixer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Głośność:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fadeoutBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Fadeout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Rozdzielczość:</string>
   </property>
  </widget>
  <widget class="QLabel" name="volLabel">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>270</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>128</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QRadioButton" name="bit8Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>8bit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bit16Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>16bit</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Reverb:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QRadioButton" name="hz2Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>22050 Hz</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hz8Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>8000 Hz</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hz4Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>44100 Hz</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hz1Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>11025 Hz</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="reverseBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reverse</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="softsfxBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Softsfx</string>
   </property>
  </widget>
  <widget class="QLabel" name="panLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>128</string>
   </property>
  </widget>
  <widget class="QLabel" name="revLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="surroundBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Surround</string>
   </property>
  </widget>
  <widget class="QSlider" name="volSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>25</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="maximum">
    <number>128</number>
   </property>
   <property name="value">
    <number>128</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="panSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>128</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>128</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="stereoBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Stereo</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="interpBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Interpolacja</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Częstotliwość próbkowania:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>83</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="chnBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Kanały :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Czas odświeżania:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="trefBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSlider" name="revSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
   <property name="pageStep">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="extEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Obsługiwane rozszerzenia plików. Wpisujesz po średniku.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Kliknij &quot;Domyślnie&quot; jeżeli będziesz chciał przywrócić domyślne ustawienia.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dExtB">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Domyślnie</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enabledB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Wtyczka załączona</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>enabledB</tabstop>
  <tabstop>interpBox</tabstop>
  <tabstop>stereoBox</tabstop>
  <tabstop>loopBox</tabstop>
  <tabstop>hqmixerBox</tabstop>
  <tabstop>surroundBox</tabstop>
  <tabstop>reverseBox</tabstop>
  <tabstop>softmusicBox</tabstop>
  <tabstop>softsfxBox</tabstop>
  <tabstop>fadeoutBox</tabstop>
  <tabstop>panSlider</tabstop>
  <tabstop>revSlider</tabstop>
  <tabstop>extEdit</tabstop>
  <tabstop>trefBox</tabstop>
  <tabstop>chnBox</tabstop>
  <tabstop>volSlider</tabstop>
  <tabstop>bit8Button</tabstop>
  <tabstop>bit16Button</tabstop>
  <tabstop>hz4Button</tabstop>
  <tabstop>hz2Button</tabstop>
  <tabstop>hz1Button</tabstop>
  <tabstop>hz8Button</tabstop>
  <tabstop>dExtB</tabstop>
  <tabstop>ApplyButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
